src/coreclr/jit/codegenloongarch64.cpp

```diff
@@ -7068,13 +7137,27 @@ void CodeGen::genCall(GenTreeCall* call)
         }
         else if (abiInfo.IsSplit())
         {
-            NYI("unimplemented on LOONGARCH64 yet");
+            assert(compFeatureArgSplit());
+            genConsumeArgSplitStruct(argNode->AsPutArgSplit());
+
+            regNumber argReg   = abiInfo.GetRegNum();
+            regNumber allocReg = argNode->AsPutArgSplit()->GetRegNumByIdx(0);
+
+            // For LA64's ABI, the split is only using the A7 and stack for passing arg.
+            assert(emitter::isGeneralRegister(argReg));
+            assert(emitter::isGeneralRegister(allocReg));
+            assert(abiInfo.NumRegs == 1);
+
+            ClassLayout* layout = argNode->AsPutArgSplit()->GetLayout(compiler);
+
+            var_types Type = (layout != nullptr) ? layout->GetGCPtrType(0) : TYP_I_IMPL;
+            inst_Mov(Type, argReg, allocReg, /* canSkip */ true);
```

Comment:
```suggestion
            regNumber allocReg = argNode->AsPutArgSplit()->GetRegNumByIdx(0);
            var_types regType  = argNode->AsPutArgSplit()->GetRegType(0);

            // For LA64's ABI, the split is only using the A7 and stack for passing arg.
            assert(emitter::isGeneralRegister(argReg));
            assert(emitter::isGeneralRegister(allocReg));
            assert(abiInfo.NumRegs == 1);

            inst_Mov(regType, argReg, allocReg, /* canSkip */ true);
```
The split node itself doesn't have layout, but does carry the register types.

Reply:
Thanks, setting it within the `lower.cpp`